LDK/resources/web/LDK/panel/SingleSubjectFilterType.js

```diff
@@ -155,7 +155,10 @@ Ext4.define('LDK.panel.SingleSubjectFilterType', {
 
     aliasTableConfig: function (subjectArray) {
         this.aliasTable.scope = this;
-        this.aliasTable.filterArray = [LABKEY.Filter.create('alias', subjectArray.join(';'), LABKEY.Filter.Types.EQUALS_ONE_OF)];
+
+        // Use contains filter to ensure case insensitivity across dbs
+        var filterType = this.caseInsensitive ? LABKEY.Filter.Types.CONTAINS_ONE_OF : LABKEY.Filter.Types.EQUALS_ONE_OF;
```

Comment:
Are we positive that an animal ID will never be a substring in another animal ID? 250 and 2501, for example?

We're probably safer switching this to send a LabKey SQL query that does a LOWER() on both sides via executeSQL() instead of forcing this into a filter for selectRows()

Reply:
it is more rare, but definitely they can be

Reply:
It's ok if it's a substring of another Id and both are returned from this query.  The results are processed after this with the user input to pick out aliases, not found or found by Id, and now if this caseInsensitive flag is set the matches are found regardless of casing.  The rest of the results are not used.  Possibly a little more payload on the query but shouldn't be significant, as noted it is rare to have substring names.

Reply:
Thanks for clarifying. I think this is worth a comment in the code too.

Reply:
Good call, I updated the comment